### PR TITLE
use dynamic route name

### PR DESCRIPTION
**Pull request by sushant12**
## What type of PR is this? (check all applicable)

- [X] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description

`redirect_to` in `users_controller.rb` was using a hardcoded route path, so I just changed it to using a dynamic route name path

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [X] no documentation needed

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ class UsersController < ApplicationController
   def edit
     unless current_user
       skip_authorization
-      return redirect_to "/enter"
+      return redirect_to new_user_registration_path_path
```
Comment (awcodify) [inline app/controllers/users_controller.rb]: can we also change the alias into `new_user_registration_path` in routes?

Comment (sushant12) [reply app/controllers/users_controller.rb]: `new_user_registration_path` has already been declared by devise. What else do you reckon? :)

Comment (awcodify) [reply app/controllers/users_controller.rb]: okay, keep it then

**Comment by sushant12** [comment]
> Thank you for this PR!.
> 
> `new_user_registration_path_path` is a bit long. Let's change that path to `sign_up_path`.

that is a good suggestion

**Pull request merged**